ports/openmvg/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "openmvg",
   "version-string": "1.6",
```

```suggestion
  "version": "1.6",
```
This might be served by [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version).